globals.d.ts

```diff
@@ -1,5 +1,78 @@
+/**
+ * @deprecated use `IS_ANDROID` from `@/env`
+ */
 declare let android: boolean;
```

Comment:
Added `@deprecated` comments to all global variables so that we get type hints while we migrate.